src/components/Integrations/IntegrationsDrawer.tsx

```diff
@@ -0,0 +1,171 @@
+    >
+      <DropdownList>
+        <DropdownItem
+          value={0}
+          key="action"
+          description={intl.formatMessage(messages.pauseDescription)}
+        >
+          {intl.formatMessage(messages.integrationdropdownPause)}
+        </DropdownItem>
+        <DropdownItem
+          value={1}
+          key="action"
+          to="#default-link2"
+          description={intl.formatMessage(messages.removeDescription)}
+        >
+          {intl.formatMessage(messages.integrationdropdownRemove)}
+        </DropdownItem>
+        <DropdownItem value={2} key="action" to="#default-link3">
+          {intl.formatMessage(messages.integrationdropdownEdit)}
+        </DropdownItem>
+      </DropdownList>
+    </Dropdown>
+  );
+};
+
+interface IntegrationsDrawerProps {
+  selectedIntegration?: IntegrationRow;
+  setSelectedIntegration: React.Dispatch<
+    React.SetStateAction<IntegrationRow | undefined>
+  >;
+}
+
+const ouiaId = 'IntegrationsTable';
+
+const IntegrationsDrawer: React.FunctionComponent<IntegrationsDrawerProps> = ({
+  selectedIntegration,
+  setSelectedIntegration,
+}) => {
+  const [activeTabKey, setActiveTabKey] = React.useState<string | number>(0);
+
+  const intl = useIntl();
+
+  const context = useDataViewEventsContext();
+
+  const handleTabClick = (
+    _event: React.MouseEvent<HTMLElement, MouseEvent>,
+    tabIndex: string | number
+  ) => {
+    setActiveTabKey(tabIndex);
+  };
+
+  useEffect(() => {
+    const unsubscribe = context.subscribe(
+      EventTypes.rowClick,
+      (integration: IntegrationRow) => {
+        setSelectedIntegration(integration);
+      }
+    );
+    return () => unsubscribe();
+  }, []);
+
+  return (
+    <DrawerPanelContent>
+      <DrawerHead>
+        <Title
+          className="pf-v5-u-mb-md"
+          headingLevel="h2"
+          ouiaId={`${ouiaId}-drawer-title`}
+        >
+          {selectedIntegration?.name}
+        </Title>
+        <DrawerActions>
+          <ActionDropdown />
+          <DrawerCloseButton
+            onClick={() => setSelectedIntegration(undefined)}
+            data-ouia-component-id={`${ouiaId}-drawer-close-button`}
+          />
+        </DrawerActions>
+      </DrawerHead>
+      <Tabs
+        activeKey={activeTabKey}
+        onSelect={handleTabClick}
+        ouiaId={`${ouiaId}-drawer-tabs`}
+      >
+        <Tab
+          eventKey={0}
+          title={
+            <TabTitleText>
+              {intl.formatMessage(messages.integrationDetailsTabTitle)}
+            </TabTitleText>
+          }
+          ouiaId={`${ouiaId}-details-tab`}
+        >
+          <IntegrationDetails integration={selectedIntegration} />
+        </Tab>
+        <Tab
+          eventKey={1}
+          title={
+            <TabTitleText>
+              {intl.formatMessage(messages.associatedEventTypesTabTitle)}
+            </TabTitleText>
+          }
+          ouiaId={`${ouiaId}-associated-event-types-tab`}
+        >
+          hi
```

Comment:
No need to show debug values in columns, let's add either null or empty string if we don't have the data available.

Reply:
ok I took it out